packages/unified-latex-util-argspec/tests/argspec-parser.test.ts

```diff
@@ -25,7 +25,7 @@ describe("unified-latex-util-argspec", () => {
         "o m o !o m",
         "!o r() m",
         "O{somedefault} m o",
-        "m e{^}",
```

Comment:
Why was this test removed?

Reply:
It was replaced with `m e^`. Now the `printRaw` function does something smarter, and it omits braces when there's a single embellishment token of single width (which is something that wasn't even accepted at pegjs grammar level before)